content/index.mdx

```diff
@@ -12,19 +12,19 @@ export default HeroLayout
     interfaces.
   </Text>
   <div>
+  <div>
```

Comment:
Adding a link to the new top level Guides section here and removing accessibility (below), as it's now inside the Guides section